integration-libs/opf/occ/adapters/occ-opf.adapter.ts

```diff
@@ -94,4 +97,39 @@ export class OccOpfAdapter implements OpfAdapter {
   protected getInitiatePaymentEndpoint(): string {
     return this.opfEndpointsService.buildUrl('initiatePayment');
   }
+
+  getVerifyPayment(
+    paymentSessionId: string,
+    payload: OpfVerifyPaymentPayload
+  ): Observable<OpfVerifyPaymentResponse> {
+    const headers = new HttpHeaders({
+      accept: 'application/json',
+      'Content-Type': 'application/json',
+      'Content-Language': 'en-us',
+      'sap-commerce-cloud-public-key':
+        this.config.opf?.commerceCloudPublicKey || '',
+    });
+
+    return this.http
+      .post<OpfVerifyPaymentResponse>(
+        this.getVerifyPaymentEndpoint(paymentSessionId),
+        JSON.stringify(payload),
+        {
+          headers,
+        }
+      )
+      .pipe(
+        catchError((error) => throwError(error)),
+        backOff({
+          shouldRetry: isJaloError,
+        }),
+        this.converter.pipeable(OPF_VERIFY_PAYMENT_NORMALIZER)
+      );
+  }
+
+  protected getVerifyPaymentEndpoint(paymentSessionId: string): string {
+    return this.opfEndpointsService.buildUrl('getVerifyPayment', {
```

Comment:
If we call ccv2 endpoints we can still use `occEndpointsService` from `@spartacus/core` to build url. Also if this one relates to the ccv2 endpoint would be good to extract this method into separate file. For example: `occ-order.adapter.ts`.

As an example please see: `opf/occ/adapters/occ-otp.adapter.ts`.
Similarly I created for OTP endpoint separate files like: `default-occ-otp-config.ts`, `occ-otp-endpoints.model.ts`. To not mixup endpoints destination and separate CC with CCv2 ones.